skrub/_datetime_encoder.py

```diff
@@ -181,7 +202,7 @@ class DatetimeEncoder(SingleColumnTransformer):
     0         2024.0             4.0          14.0            1.713053e+09
     1         2024.0             5.0          15.0            1.715731e+09
     >>> encoder.extracted_features_
-    ['year', 'month', 'day', 'total_seconds']
+    ['birthday_year', 'birthday_month', 'birthday_day', 'birthday_total_seconds']
```

Comment:
I think this should stay as before (feature description without the column name) , see https://github.com/skrub-data/skrub/pull/1235/files#r1954645619